service/create/namespace.go

```diff
@@ -1,24 +1,24 @@
 package create
 
 import (
-	"github.com/giantswarm/awstpr"
+	"github.com/giantswarm/clustertpr"
 	microerror "github.com/giantswarm/microkit/error"
 	"k8s.io/client-go/pkg/api/errors"
 	"k8s.io/client-go/pkg/api/unversioned"
 	"k8s.io/client-go/pkg/api/v1"
 )
 
-func (s *Service) createClusterNamespace(cluster awstpr.CustomObject) error {
+func (s *Service) createClusterNamespace(cluster clustertpr.Cluster) error {
 	namespace := v1.Namespace{
 		TypeMeta: unversioned.TypeMeta{
 			Kind:       "Namespace",
 			APIVersion: "v1",
 		},
 		ObjectMeta: v1.ObjectMeta{
-			Name: cluster.Name,
+			Name: cluster.Cluster.ID,
 			Labels: map[string]string{
-				"cluster":  cluster.Name,
-				"customer": cluster.Spec.Customer.ID,
+				"cluster":  cluster.Cluster.ID,
+				"customer": cluster.Customer.ID,
```

Comment:
I think that it'd be cool to put those methods in `clustertpr`/`awstpr`/`kvmtpr` repos.